### ISSUE TITLE
mltshp logo should stay the same color when clicked (or change more dramatically)

### ISSUE THREAD

**Issue by jessamynwest** [MLTSHP/mltshp-patterns#268]
The MLTSHP logo is a slightly different color when clicked. If it’s an indicator of something it’s maybe not a good one, and if it’s not an indicator, we should make it stay the same color.

**Comment by spaceninja** [comment]
This was a first pass at an focus indicator. We need a focus indictor for accessibility, but perhaps changing colors is the wrong way to handle it. Maybe there's a way to add a focus ring that wraps the shape of the logo instead of making a box shape around the logo?

**Comment by jessamynwest** [comment]
Yeah that is a good idea, or a more obvious change I think would also be fine. I think this is too subtle, for a while I felt like it was just my fl.ux night shift.

**Comment by ardgedee** [comment]
I think we should leave the logo without state cues. Working on appropriate tagging in the HTML would be better.

**Comment by spaceninja** [comment]
I'm not sure what you mean by appropriate tagging, but the logo is an interactive element, and as such, [requires a focus state for accessibility](https://www.deque.com/blog/give-site-focus-tips-designing-usable-focus-indicators/).

Since we've disabled the default browser focus ring, we need to provide a replacement.

**Comment by bradchoate** [comment]
Is there any way to condition CSS rules based on accessibility _needs_?

**Comment by spaceninja** [comment]
Sort of — the upcoming `focus-visible` selector lets us target focus states at keyboard users. Browser support isn't great right now, but should be improving rapidly. I could try that in the fix for this.

**Comment by wjcstp** [comment]
I believe those ARIA tags and focus states like this are also used for
screen readers, so would go a long way toward accessibility, however we're
displaying it on-screen. Having the correct tagging in place allows folks
with specific local style changes to improve their experience.

Is the objection to the default browser focus ring just aesthetic?

Bill Connell
St. Paul, MN
